trade.lic

```diff
@@ -1,78 +1,419 @@
 =begin
   Documentation: https://elanthipedia.play.net/Lich_script_repository#trade
 =end
-custom_require.call(%w[common common-items common-money common-travel drinfomon equipmanager events])
+custom_require.call(%w[common common-items common-arcana common-crafting common-money common-travel drinfomon equipmanager events])
 
 class Trade
   include DRC
+  include DRCA
   include DRCT
   include DRCI
+  include DRCC
   include DRCM
 
   def initialize
     @town_data = get_data('town')
     @equipment_manager = EquipmentManager.new
     @settings = get_settings
     @contract_container = @settings.trade_contract_container
-    @hometown = @settings.hometown
-    bput("open my #{@contract_container}", 'You open', 'That is already')
-    @equipment_manager.empty_hands
+    @hub_city = 'Crossing'
+    @emergency_delivery_town = nil
+    @current_caravan_destination = nil
+    @caravan_coins_on_hand = @settings.caravan_coins_on_hand.to_i || 25555
+    @caravan_interior = @settings.caravan_interior || false
```

Comment:
I would add this to base with the default as false.

Reply:
Yep, that was the intention -- I didn't look closely enough at base when I was making them line up.  Thanks for catching these!  I'll do another similar run-through shortly

Reply:
Sweet, just trying to give you feedback from a fresh pair of eyes perspective :)

Reply:
I'm never displeased to see people catching my mistakes or suggesting a different way to do things.  Definitely a lot better than after it goes live.